base_station/gui/src/components/ESTask.vue

```diff
@@ -0,0 +1,398 @@
+<template>
+  <div class="wrapper">
+    <div class="box header">
+      <img src="/static/mrover.png" alt="MRover" title="MRover" width="48" height="48" />
+      <h1>ES Dashboard</h1>
+      <div class="spacer"></div>
+      <div class="comms">
+        <ul id="vitals">
+          <li><CommIndicator v-bind:connected="connections.websocket" name="Web Socket" /></li>
+          <li><CommIndicator v-bind:connected="connections.lcm" name="Rover Connection Status" /></li>
+          <li><CommIndicator v-bind:connected="connections.motors && connections.lcm" name="Driving" /></li>
+        </ul>
+      </div>
+      <div class="spacer"></div>
+      <div class="help">
+        <img src="/static/help.png" alt="Help" title="Help" width="48" height="48" />
+      </div>
+      <div class="helpscreen"></div>
+      <div class="helpimages" style="display: flex; align-items: center; justify-content: space-evenly">
+        <img v-if="controlMode === 'arm'" src="/static/arm.png" alt="Robot Arm" title="Robot Arm Controls" style="width: auto; height: 70%; display: inline-block" />
+        <img v-else-if="controlMode === 'soil_ac'" src="/static/soil_ac.png" alt="Soil Acquisition" title="Soil Acquisition Controls" style="width: auto; height: 70%; display: inline-block" />
```

Comment:
TBH I copied it from the old RATask. I can remove it.

Reply:
Honestly we should either remove all the help images or make sure they're up to date. I don't think we've touched them in a long time because almost no one knows they exist.